pkg/server/handlers.go

```diff
@@ -68,16 +71,17 @@ func Validation(clients *clients.Clients) ([]admission.ValidatingAdmissionHandle
 		roles := role.NewValidator()
 		rolebindings := rolebinding.NewValidator()
 		setting := setting.NewValidator(clients.Management.Cluster().Cache(), clients.Management.Setting().Cache())
```

Comment:
I'm not big on local variables shadowing package names, but I see here that they're just being used to avoid coming up with a very long 1-liner. And the assignment to `token` is the only change in this PR. How about a compromise, call it `tokenValidator`, and we'll call it even.

Reply:
Shadowing is part of day-to-day life with Go and package shadowing is the least problematic type. But I agree here it could've been avoided simply by going with the plural `settings := ...`.
One the other hand that one liner is much better IMHO when broken into separate lines because that `append` that follows is already out-of-hand and resource packages are self explanatory.